@ehsavoie CI fails due to 401s when pulling the image:

{code}
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=quay.io/repository/arkmq-org/activemq-artemis-broker:artemis.2.42.0, imagePullPolicy=DefaultPullPolicy(), imageNameSubstitutor=org.testcontainers.utility.ImageNameSubstitutor$LogWrappedImageNameSubstitutor@6d293993)
	at org.testcontainers.containers.GenericContainer.getDockerImageName(GenericContainer.java:1364)
	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:351)
	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:322)
	at org.wildfly.test.integration.microprofile.reactive.RunArtemisAmqpSetupTask.setup(RunArtemisAmqpSetupTask.java:87)
	... 103 more
Caused by: com.github.dockerjava.api.exception.UnauthorizedException: Status 401: {"message":"unauthorized: access to the requested resource is not authorized"}
{code}

--- a/testsuite/integration/expansion/src/test/java/org/wildfly/test/integration/microprofile/reactive/RunArtemisAmqpSetupTask.java
+++ b/testsuite/integration/expansion/src/test/java/org/wildfly/test/integration/microprofile/reactive/RunArtemisAmqpSetupTask.java
@@ -54,8 +54,7 @@ public class RunArtemisAmqpSetupTask implements ServerSetupTask {
     @Override
     public void setup(ManagementClient managementClient, String containerId) throws Exception {
         try {
-
-            DockerImageName imageName = DockerImageName.parse("quay.io/artemiscloud/activemq-artemis-broker:1.0.32");
+            DockerImageName imageName = DockerImageName.parse("quay.io/arkmq-org/activemq-artemis-broker:artemis.2.42.0");
             container = new GenericContainer<>(imageName);
             container.addExposedPort(AMQP_PORT);
             container.withEnv(Map.of(

